convert-mention.jsonが見つからず実行に失敗する

# 問題

こちらのactionを利用してメンションを飛ばそうとしたところ下記のログがactionに流れて失敗した
```
Cannot find module '/home/runner/work/repo/repo/.github/convert-mention.json'
```

# 状態の共有
## ディレクトリ階層
```
.github
├── convert-mention.json
└── workflows
    └── github2slack.yml
```

## github2slack.yml
```yml
name: Convert GitHub's mention to Slack's

on:
  issue_comment:
    types: [created, edited]
  pull_request_review_comment:
    types: [created, edited]
  pull_request_review:
    types: [submitted, edited, dismissed]
  pull_request:
    types: [review_requested, created, edited]

jobs:
  message:
    runs-on: ubuntu-latest

    steps:
      - name: Convert mention
        uses: tyankatsu0105/convert-mention-action@v2.0.0
        env:
          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
          SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
```

## convert-mention.json
```json
{
  "github user1": "slack user1",
  "github user2": "slack user2"
}
```

# その他
issueの投げ方雑でごめんなさい🙏

@tyankatsu0105 
convert-mentionの前にcheckoutをいれてみました。

```
Run tyankatsu0105/convert-mention-action@v2.0.0
  with:
    users: .github/convert-mention.json
  env:
    SLACK_WEBHOOK_URL: ***
    SLACK_TOKEN: ***
##[error]Cannot read property 'filter' of undefined
```

https://github.com/tyankatsu0105/convert-mention-action/blob/58d42c92d094948153faf677716c831155ecc4e9/src/util/getMatchedMembers.ts#L36-L40
ここのfilterが該当部分だとおもいますが、memberが取得できていない？

--- a/README.md
+++ b/README.md
@@ -80,9 +80,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - user: actions/checkout@v2
+
       - name: Convert mention
         uses: tyankatsu0105/convert-mention-action@v2.0.0
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
           SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
 ```
+
+>This action requests the user:read function and permission scope of the slack app.

